Foundation/NSLog.swift

```diff
@@ -41,3 +41,11 @@ public func NSLog(_ format: String, _ args: CVarArg...) {
         NSLogv(format, $0) 
     }
 }
+
+// hook for System.out.println()
+#if os(Android)
+public var debug = {
```

Comment:
I agree with @phausler - please no additional public symbols, and especially not one named `debug`.